{"timestamp":"2025-09-07T14:13:16.015989","level":"info","event":"DAG bundles loaded: dags-folder, example_dags","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager"}
{"timestamp":"2025-09-07T14:13:16.016557","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/pipeline.py","logger":"airflow.models.dagbag.DagBag"}
{"timestamp":"2025-09-07T14:13:16.067811Z","level":"info","event":"Task instance is in running state","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-07T14:13:16.068211Z","level":"info","event":" Previous state of the Task instance: TaskInstanceState.QUEUED","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-07T14:13:16.068488Z","level":"info","event":"Current task name:import_data","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-07T14:13:16.068788Z","level":"info","event":"Dag name:extract_and_store_retail_data","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-07T14:13:16.070260","level":"error","event":"Task failed with exception","logger":"task","error_detail":[{"exc_type":"FileNotFoundError","exc_value":"[Errno 2] No such file or directory: 'data/Online_Retail.csv'","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":920,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1215,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":397,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":216,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":239,"name":"execute_callable"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/callback_runner.py","lineno":81,"name":"run"},{"filename":"/opt/airflow/dags/pipeline.py","lineno":20,"name":"import_data_and_transform"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/pandas/io/parsers/readers.py","lineno":948,"name":"read_csv"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/pandas/io/parsers/readers.py","lineno":611,"name":"_read"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/pandas/io/parsers/readers.py","lineno":1448,"name":"__init__"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/pandas/io/parsers/readers.py","lineno":1705,"name":"_make_engine"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/pandas/io/common.py","lineno":863,"name":"get_handle"}],"is_group":false,"exceptions":[]}]}
{"timestamp":"2025-09-07T14:13:16.071595Z","level":"info","event":"Task instance in failure state","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-07T14:13:16.072399Z","level":"info","event":"Task start","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-07T14:13:16.072698Z","level":"info","event":"Task:<Task(PythonOperator): import_data>","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-07T14:13:16.072956Z","level":"info","event":"Failure caused by [Errno 2] No such file or directory: 'data/Online_Retail.csv'","chan":"stdout","logger":"task"}
